pkgs/development/libraries/webp-pixbuf-loader/default.nix

```diff
@@ -52,6 +52,8 @@ stdenv.mkDerivation rec {
     GDK_PIXBUF_MODULEDIR="$out/${moduleDir}" \
     gdk-pixbuf-query-loaders --update-cache
 
+    # gdk-pixbuf disables the thumbnailer in cross-builds.
```

Comment:
How did you determine this? I see no such thing in https://github.com/aruiz/webp-pixbuf-loader/blob/0.2.7/meson.build

Reply:
It's disabled in **gdk-pixbuf** not webp-pixbuf-loader. Checkout this: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/master/meson.build?ref_type=heads#L397

Reply:
Hmm, [indeed](https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/commit/fc37708313a5fc52083cf10c9326f3509d67701f). That makes little sense to me. The only thing that could be preventing thumbnailer from working is not generating the `loaders.cache` but we are creating that ourselves using emulator:

https://github.com/NixOS/nixpkgs/blob/e6719289cdd68c290fa076a6409a2d89535ade5e/pkgs/development/libraries/gdk-pixbuf/default.nix#L123-L124

But I would assume this will be issue in cross here as well, yet you are keeping the line above.

Also, since we do install the `.thumbnailer` file unconditionally in this package, the thumbnailer will not be fully disabled – I believe it will just fail to be discovered at runtime.

Reply:
So what are your suggested next steps? I tried enabling the thumbnailer executable in their buildsystem but this will need a more complex patch inside their buildsystem since this executable also has dependencies on `loaders.cache`. So disabling the wrapper here seemed more straightforward to me.

Reply:
If this workaround fixes cross for you, it is good enough for me, the change is thankfully unobtrusive. I am just surprised `gdk-pixbuf-query-loaders` above works in cross.

Would be nice to include more detailed explanation of what exactly is broken in the commit message (e.g. `makeWrapper` expects the wrapped file to exists) and limitations (keeping the thumbnailer file pointing to non-existent `gdk-pixbuf-thumbnailer-webp` but it is okay since thumbnailer machinery will disable `thumnailer` files with non-executable paths in `Exec` key).

Reply:
Done.